### PR TITLE
Issue #11292: icon for GUI

### PR DESCRIPTION
**Pull request by pbludov**
Issue #11292 

The icon image is same as `checkstyle_logo_small_64.png` from the site source.

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/gui/Main.java [hunk 1]
```diff
@@ -47,6 +49,8 @@ public static void main(final String... args) {
                 final File sourceFile = new File(args[0]);
                 mainFrame.openFile(sourceFile);
             }
+            final URL iconUrl = Main.class.getResource("icon.png");
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/gui/Main.java:52]: all images are at https://github.com/checkstyle/resources/tree/master/img/cs-logos-twitter-gplus-backgrounds and in other folders of this repo.
https://github.com/checkstyle/resources/blob/master/img/cs-logos-twitter-gplus-backgrounds/cs-120x120.png is better as CS and pancil are in same size.


and we might figureout from where you got this page and fix it also

**Review by romani — CHANGES_REQUESTED** [review]
item:

File: src/main/java/com/puppycrawl/tools/checkstyle/gui/Main.java (hunk 1 above)
Comment (pbludov) [reply src/main/java/com/puppycrawl/tools/checkstyle/gui/Main.java]: done

**Review by romani — APPROVED** [review]

**Comment by pbludov** [comment]
CI is failing. This is a false positive. Spell checker should not process PNG files.

**Pull request merged**